scripts/claude-stub.py

```
@@ -0,0 +1,314 @@
+    return lines
+
+
+def report(lines: list[str]) -> None:
+    for line in lines:
+        print(f"claude-stub: {line}", file=sys.stderr)
+
+
+def run_claude(binary: str, claude_args: list[str], env: dict[str, str]) -> int:
+    """Run claude with stdio inherited; Ctrl-C reaches it, we still summarize."""
+    try:
+        child = subprocess.Popen([binary, *claude_args], env=env)
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)